bin/metrics-elb.rb

```diff
@@ -106,8 +106,8 @@ def cloud_watch_metric(metric_name, value, load_balancer_name)
         }
       ],
       statistics: [value],
-      start_time: config[:end_time] - config[:period],
-      end_time: config[:end_time],
+      start_time: config[:end_time] - config[:fetch_age] - config[:period],
```

Comment:
Based on this it seems like we should rename `fetch_age` to `start_time` or am I missing something?

Reply:
It's not really the start time, it's the duration in the past you want to get the metrics from.

It actually makes more sense than having a ``start_time`` flag:

* using a "fetch age" flag, you don't have to specify the end time and can use a fixed "fetch age" duration to say: every time the check executes, get me the metrics from the last 60 seconds for example:
```
metrics-elb.rb --fetch_age 60s
```
* using a "start time" flag, you would need to specify a specific time which would need to be updated every time you are executing the check (otherwise you get the same metrics all the times), which is not especially practical in the context in which Sensu works.

```
metrics-elb.rb --start-time 2018-02-06T17:45:15+01:00
```

Does it make more sense?